src/app/results/results.component.html

```diff
@@ -75,7 +75,7 @@
                 <ul class="pagination" id="pag-bar">
                     <li class="page-item"><span class="page-link" href="#" (click)="decPresentPage()">Previous</span></li>
                     <li class="page-item" *ngFor="let num of getNumber(maxPage)"><span class="page-link" *ngIf="presentPage>=4 && num<=noOfPages" [class.active_page]="getStyle(presentPage-3+num)"
-                            (click)="getPresentPage(presentPage-3+num)" href="#">{{presentPage-3+num}}</span>
+                            (click)="getPresentPage(presentPage-4+num)" href="#">{{presentPage-3+num}}</span>
```

Comment:
If you mean I should change the function, I cannot do that, since it seems to work properly for  pages < 4 (see lines 79-80), and that would stop working properly.